llvm/test/SYCLLowerIR/lower_kernel_props.ll

```diff
@@ -38,7 +48,22 @@ define weak_odr dso_local spir_kernel void @__large_grf_kernel2() #0 !sycl_expli
   ret void
 }
 
+; -- This kernel calls the marker function indirectly
+define weak_odr dso_local spir_kernel void @__auto_grf_kernel1() !sycl_explicit_simd !0 !intel_reqd_sub_group_size !1 {
+; CHECK: {{.*}} spir_kernel void @__auto_grf_kernel1() #1
```

Comment:
In general, it is better not to hardcode attribute group IDs, because they tend to change as compiler evolves and sometimes cause issues in downstream due to different passes pipelines.

Reply:
yeah sorry i copy and pasted part of the test which was doing that and i shouldnt have read it more, will fix